packages/inference/src/snippets/getInferenceSnippets.ts

```diff
@@ -1,14 +1,30 @@
 import type { PipelineType, WidgetType } from "@huggingface/tasks/src/pipelines.js";
 import type { ChatCompletionInputMessage, GenerationParameters } from "@huggingface/tasks/src/tasks/index.js";
-import { type InferenceSnippet, type ModelDataMinimal, getModelInputSnippet } from "@huggingface/tasks";
+import {
+	type InferenceSnippet,
+	type InferenceSnippetLanguage,
+	type ModelDataMinimal,
+	inferenceSnippetLanguages,
+	getModelInputSnippet,
+} from "@huggingface/tasks";
 import type { InferenceProvider, InferenceTask, RequestArgs } from "../types";
 import { Template } from "@huggingface/jinja";
 import { makeRequestOptionsFromResolvedModel } from "../lib/makeRequestOptions";
 import fs from "fs";
 import path from "path";
 import { existsSync as pathExists } from "node:fs";
 
-const TOOLS = ["huggingface_hub", "requests", "fal_client", "openai"];
+const PYTHON_CLIENTS = ["huggingface_hub", "fal_client", "requests", "openai"];
+const JS_CLIENTS = ["fetch", "huggingface.js", "openai"];
+const SH_TOOLS = ["curl"];
```

Comment:
For consistency with the other languages
```suggestion
const SH_CLIENTS = ["curl"];
```